common/frame.go

```diff
@@ -1966,15 +1958,15 @@ type frameExecutionContext interface {
 }
 
 func (f *Frame) runActionOnSelector(
-	ctx context.Context, selector string, strict bool, fn elementHandleActionFunc,
+	ctx context.Context, selector string, strict bool, fn elementHandleActionFunc, nfn func() bool,
```

Comment:
What does `nfn` mean here? Maybe we can make the parameter name more explicit.

Reply:
Resolved in 869847bcd1e43e4516eee6e836fa3249e54f4e11